app/pods/components/agenda/agenda-header/component.js

```diff
@@ -60,6 +64,12 @@ export default Component.extend(FileSaverMixin, {
     return true;
   }),
 
+  amountOfAgendaitemsNotFormallyOk: computed('isApprovingAllAgendaitems', async function() {
```

Comment:
De "dependents" van een computed zijn gewoonlijk dingen die gebruikt worden bij de berekening van de computed value ... Hier eerder `currentAgenda.@each.formallyOk` dus? (Zie [de ember guides](https://guides.emberjs.com/v3.8.0/object-model/computed-properties-and-aggregate-data/#toc_each))

Reply:
Dit doet dan niet de verwachte change, ik heb het gevoel dat currentAgenda niet doet wat we verwachten wat het doet.. probeer maar eens die lijn te veranderen en dan alles formeel goedkeuren, en vervolgens klik je nog eens op `alles formeel goedkeuren`, je gaat zien dat je nog steeds hetzelfde aantal elementen wilt aanpassen terwijl dit niet kan.

Er zit mijninzichts iets grondig fout in de bestaande stukken/computed properties, we zouden daar naar moeten kijken.

Reply:
Fixed.